src/frontend/src/lib/styles/global/links.scss

```diff
@@ -1,6 +1,12 @@
+@import '../theme/base-colors.scss';
+
 a.no-underline {
 	&:hover,
 	&:focus {
 		text-decoration: none;
 	}
 }
+
+a.blue-link {
+	color: #{$brandeis-blue};
```

Comment:
is there a way to use tailwind directly in the hyperlink? or at least to use the color variables like `var(--color-brand-base)`?

Reply:
I tried, but it's not possible. I'm afraid to use color-brand-base for the reason that the link should always be blue, and if we change the brand color to yellow (as an example), the links would be yellow..